### ISSUE TITLE
feat: list and revoke invites?

### ISSUE THREAD

**Issue by robsimmons** [letsdiscodev/cli#43]

**Comment by gregsadetsky** [comment]
good idea! the invites do expire within ~24 hours ([here](https://github.com/letsdiscodev/disco-daemon/blob/main/disco/utils/apikeyinvites.py#L19)), but yes, for completeness would be good thanks!
